### DISCUSSION TITLE
routes - dest issue with nested folders

### DISCUSSION THREAD

**Discussion by obayesshelton** [vercel/community#3877] [Help]
### Summary

Hi, 

My vercel.json looks like the following:

```
"routes": [
        {
            "src": "/(css|js|vendor|assets|build)/(.*)",
            "dest": "public/$1/$2"
        }
    ],
```

However it doesn't allow me to access a nested `.css` file such as `public/vendor/lunar/admin-hub/app.css` I think it down to the dest mapping but I cannot really figure out it. 

The final URL would be ``https://branch-vyblee-2-0.vercel.app/vendor/lunar/admin-hub/app.css``

Could somebody help please. 

### Steps to Reproduce

Use the routes section in oyur vercel.json and have a CSS file in the path `public/vendor/lunar/admin-hub/app.css` and go to the URL `https://branch-vyblee-2-0.vercel.app/vendor/lunar/admin-hub/app.css`

**Comment by amyegan** [comment]
Hey @obayesshelton. It seems like you have a bigger problem with routing in your app. I also get a 404 at the root URL https://branch-vyblee-2-0.vercel.app

If you share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), that will give me a better view of what is going wrong.
